brave/brave-core
# Update adblock licenses

Resolves brave/brave-browser#29346



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Open `brave://credits`
2. Scroll down to "Brave Ad Block" and click "show license"
3. Check that the list includes an entry for (note: the list is **not** sorted alphabetically by _Name_):
```
Name: Blocklists Anti-porn list
URL: https://github.com/blocklistproject/Lists/blob/master/adguard/porn-ags.txt
License: Unlicense
```
